README.rst

```diff
@@ -146,6 +150,16 @@ Conda (Anaconda)
 
 Currently, it supports only python version 2.7, 3.6 and 3.7.
 
+Snap (Snapcraft)
+----------------
+
+.. code-block:: bash
+
+   snap install dvc
```

Comment:
So no additional flags needed for now? I'm completely lost with that `classic` approval. Is this even going to work without that approval?

Reply:
added some explicit commands which users would need to run to grant permission (`classic` would grant everything)

Reply:
I haven't tested all functionality so there's likely a few permissions missing...

Reply:
I just did all of https://dvc.org/doc/tutorials/versioning and only got stuck at the end with `dvc run` since with `strict` confinement there's no way to access host system binaries (including e.g. `python`). Not something we can fix without `--classic`...

Reply:
Isn't there a way to explicitly tell snap to install with "classic" permissions even if it is dangerous?

Reply:
the snap won't be auto-downloadable (`snap install --dangerous --classic dvc` won't work). Users would have to download the snap file from the pending approval section (might even need logging in?) or from github releases and then `snap install --dangerous --classic <dvc_VER.snap>`

Reply:
Maybe the latter is the only feasible option for now.